tests/unit_grpc/test_grpc_index_describe_index_stats.py

```diff
@@ -0,0 +1,37 @@
+from copy import deepcopy
+
+import numpy as np
+import pandas as pd
+import pytest
+
+from pinecone import Config
+from pinecone.grpc import GRPCIndex
+from pinecone import DescribeIndexStatsRequest
+from pinecone.core.grpc.protos.vector_service_pb2 import (
+    DescribeIndexStatsRequest,
+)
+from pinecone.grpc.utils import dict_to_proto_struct
+
+
+class TestGrpcIndexDescribeIndexStats:
+    def setup_method(self):
+        self.config = Config(api_key="test-api-key", host="foo")
+        self.index = GRPCIndex(config=self.config, index_name="example-name", _endpoint_override="test-endpoint")
+
+    def test_describeIndexStats_callWithoutFilter_CalledWithoutFilter(self, mocker):
```

Comment:
**nitpick:** I was going to ask about the duplicative seeming naming for tests in the grpc suite, but it seems to be across the board in some of these files.